pkg/helm/client.go

```diff
@@ -360,7 +360,7 @@ func (h *Client) content(ctx context.Context, req *rls.GetReleaseContentRequest)
 
 // Executes tiller.GetVersion RPC.
 func (h *Client) version(ctx context.Context, req *rls.GetVersionRequest) (*rls.GetVersionResponse, error) {
-	c, err := grpc.Dial(h.opts.host, grpc.WithInsecure())
+	c, err := grpc.Dial(h.opts.host, grpc.WithInsecure(), grpc.WithTimeout(5))
```

Comment:
1. From the grpc godoc: ```WithTimeout returns a DialOption that configures a timeout for dialing a ClientConn initially. This is valid if and only if WithBlock() is present.``` ...I.e. Add grpc.WithBlock() in addition to the timeout.

2. The default variant of time.Duration is Nanosecond. If "5 Seconds" is desired, then this should be ```grpc.WithTimeout(5*time.Second)```